tests/test-output-flex-schema.cpp

```diff
@@ -31,12 +31,12 @@ TEST_CASE("config with schema should work")
 
     REQUIRE(1362 == conn.get_count("myschema.osm2pgsql_test_point"));
 
-    REQUIRE(1 ==
+    REQUIRE(0 ==
             conn.get_count("pg_proc",
                            "proname = 'osm2pgsql_test_point_osm2pgsql_valid'"));
 
-    REQUIRE(1 == conn.get_count("pg_trigger"));
-    REQUIRE(1 ==
+    REQUIRE(0 == conn.get_count("pg_trigger"));
+    REQUIRE(0 ==
```

Comment:
So you don't test at all anymore that the trigger ends up in the right schema?

Reply:
Ups. Using the linestring table now fixes this.